src/lucky/request_type_helpers.cr

```diff
@@ -91,6 +91,13 @@ module Lucky::RequestTypeHelpers
     request.headers["X-Requested-With"]?.try(&.downcase) == "xmlhttprequest"
   end
 
+  # Check if the request is multipart
+  #
+  # This tests if the `Content-Type` header is `multipart/form-data`
+  def multipart? : Bool
+    request.headers["Content-Type"]?.try(&.downcase) == "multipart/form-data"
```

Comment:
I believe that most content-types won't be just `multipart/form-data`. Generally these will look more like

```
multipart/form-data; boundary=------boundary123
```

What I had to do in a recent commit was use `starts_with?` https://github.com/luckyframework/breeze/blob/master/src/breeze/actions/mixins/action_helpers.cr#L76-L78

Reply:
Thanks 👍 I read a little more about it and I see now how multipart is working 🎉